external-crates/move/move-compiler/src/editions/mod.rs

```diff
@@ -29,6 +29,7 @@ pub struct Edition {
 pub enum FeatureGate {
     PublicPackage,
     PostFixAbilities,
+    PublicStructTypeVisibility,
```

Comment:
Nit, maybe
```suggestion
    StructTypeVisibility,
```
Since that is more what this about, not just public